pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
```

Comment:
no need to add a new file. see https://docs.pytest.org/en/6.2.x/customize.html#setup-cfg

Reply:
However, here is a warning. 

```
Warning
Usage of setup.cfg is not recommended unless for very simple use cases. .cfg files use a different parser than pytest.ini and tox.ini which might cause hard to track down problems. When possible, it is recommended to use the latter files, or pyproject.toml, to hold your pytest configuration.
```

Reply:
I thought ours belong to the simple case haha. Anyway keep or not, either is okay to me.